service-springboot-rest/src/main/java/org/keycloak/quickstart/springboot/ProductServiceApplication.java

```diff
@@ -36,12 +35,10 @@
 	protected ServletContextListener listener() {
 		return new ServletContextListener() {
 
-			@Override
 			public void contextInitialized(ServletContextEvent sce) {
 				logger.info("ServletContext initialized");
 			}
 
-			@Override
```

Comment:
Why remove this?

Reply:
This is how the default Intellij formatter behaves (since the class contained a mixture of tabs and spaces I reformatted the whole file).

Reply:
Removing @Override here makes no sense